### PR TITLE
chore: update default geo-coordinates for algolia models

### PR DESCRIPTION
**Pull request by francotobares**
chore: update default geo-coordinates for algolia models

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @francotobares!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/openedx/.github/blob/master/CONTRIBUTING.md)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

    
Once you've signed the CLA, please allow 1 business day for it to be processed.  After this time, you can re-run the     CLA check by editing the PR title.  If the problem persists, you can tag the `@openedx/cla-problems` team in a     comment on your PR for further assistance.

File: course_discovery/apps/course_metadata/algolia_models.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 # Every record needs geolocation data to show up in results after we turn on georanking.
 # These are the coordinates of the HQ office in Maryland
```
Comment (bseverino) [inline course_discovery/apps/course_metadata/algolia_models.py]: nit: change this comment

**Review by bseverino — APPROVED** [review]

**Review by Ali-D-Akbar — COMMENTED** [review]
Assuming the coordinates are correct and the inline-comment is updated, this is good to merge.

**Review by Ali-D-Akbar — APPROVED** [review]

**Review by AfaqShuaib09 — COMMENTED** [review]
Make sure to update default geolocations coordinates in `test_algolia_models.py`.

- https://github.com/openedx/course-discovery/blob/master/course_discovery/apps/course_metadata/tests/test_algolia_models.py#L463-L468

- https://github.com/openedx/course-discovery/blob/master/course_discovery/apps/course_metadata/tests/test_algolia_models.py#L782-L787

Additionally, please update the commit message following commit-lint conventions. Commit type would be `chore`

**Review by AfaqShuaib09 — APPROVED** [review]
Squash commits before merge.
please update commit message following the below listed format otherwise it will raise CI commitlint error.
like `commit-type: commit message`
In your case it would be like
`git commit -m 'chore: update default geo-coordinates for algolia models'`

**Comment by e0d** [comment]
@francotobares I've re-run the tests.  I doesn't look like we have a Contributor Licenses Agreement on file for you.  Are you contributing this as an individual or as work for an organization?

**Comment by e0d** [comment]
I notice there are some commit-lint failures.  Please note that we use conventional commits across Open edX projects.  You can read about the details [here](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html#type).  Can you please amend your commit messages to follow our standard?

**Comment by francotobares** [comment]
> @francotobares I've re-run the tests. I doesn't look like we have a Contributor Licenses Agreement on file for you. Are you contributing this as an individual or as work for an organization?

Hii work for edx/2u it part of this ticket https://2u-internal.atlassian.net/browse/WS-4108?atlOrigin=eyJpIjoiYmE2ODQ0NDg0YzgyNDJhMmFjMzE1ODE1Y2QyZTE4MWMiLCJwIjoiaiJ9

**Comment by mphilbrick211** [comment]
Hi @francotobares! Could you please have your manager reach out to us at [oscm@axim.org](mailto:oscm@axim.org) to add you to the edX/2U entity CLA?

**Comment by e0d** [comment]
@francotobares also please look at the commit lint docs that I linked above.

**Comment by ashley3** [comment]
I believe this is ready for re-review / testing :)

**Comment by Ali-D-Akbar** [comment]
Squash commits + rename your commit by following [OEP-51](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html) commit messages so I can merge this. Thanks.

**Comment by ashley3** [comment]
@Ali-D-Akbar looks like it's ready to merge now 😄

**Pull request merged**